pkg/api/server.go

```diff
@@ -318,9 +370,13 @@ func (s *Server) Run(ctx context.Context) error {
 	logger.Infof("API server listening on %s", host)
 
 	mux := http.NewServeMux()
-	mux.HandleFunc("/", get(s.handleFaultsRequest))
+	mux.HandleFunc("/", get(s.handleRoot))
```

Comment:
this is here as it facilitates watching the API output while developing stuff

I think eventually we will want to have a nice landing page but in the mean time I think this is a strict regression in functionality so I'd suggest we don't make this change to the `/` path

Reply:
ah gotcha, that sgtm thanks for pointing this out